guide/14-deep-learning/object-detection.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>"Once the later layers have been sufficiently trained, the earlier layers are <code>unfreeze()</code> and fine-tuned to the" -&gt; "... the earlier layers are unfrozen (by calling <code>unfreeze()</code>) and ..."</p><p> </p><p>"Please note that this step is only optional." -&gt; remove the word 'only'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>